packages/manager/apps/dedicated/client/app/dedicated/server/terminate/dedicated-server-terminate.html

```diff
@@ -16,6 +16,14 @@
                 <oui-spinner></oui-spinner>
             </div>
 
+            <div class="mb-3" data-ng-if="ctrl.terminationUnavailable">
+                <oui-message data-type="error">
```

Comment:
Why not prefer applied the`ngIf` directive directly on the `<oui-message` component?

Reply:
This is done :)